Homework6/AttributeLibrary/Logger.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Reflection;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using System.IO;
+
+namespace AttributeLibrary
+{
+    public class Logger
+    {
+        public string FileName { get; }
+
+        public Logger(string fileName)
+        {
+            FileName = fileName;
+        }
+
+        public void Track(object testObject)
+        {
+            var objType = testObject.GetType(); //get passed object type
+
+            Dictionary<string, string> PropsAndValues = new(); // initialize dictionary for (property name : property value) pairs
+
+            PropertyInfo[] props = objType.GetProperties(); //get an array of properties for passed object type
```

Comment:
minor perf update - call GetProperties after check that TrackingEntityAttribute is set. 
Because if it does not set, GetProperties was called for nothing.